pyccel/parser/syntax/openmp.py

```diff
@@ -75,7 +75,7 @@ def expr(self):
             else:
                 raise TypeError('Wrong clause for OmpParallelConstruct')
 
-        return AnnotatedComment('omp', txt)
+        return OMP_ParallelConstruct('omp', txt)
```

Comment:
It seems odd to me to have `omp` in the name but to still need to pass `'omp'` as an argument. Presumably this is due to the constructor of `AnnotatedComment`. Couldn't the constructor of `OMP_ParallelConstruct` be modified to only require one argument and pass `'omp'` to `AnnotatedComment` itself?

Reply:
i agree, if the `AnnotatedComment` class is not used for `OpenACC` then we should modify it to only require one argument.

Reply:
I suspect `AnnotatedComment` might be used for `OpenACC`. That is why my suggestion is not to modify `AnnotatedComment`, but just `OMP_ParallelConstruct`
I.e:
```
class OMP_ParallelConstruct:
    def __new__(cls, txt):
        return AnnotatedComment.__new__(cls, 'omp', txt)
```